### PR TITLE
Release 1.24.0

### PR DESCRIPTION
**Pull request by KTibow**
Depends on https://github.com/PrismarineJS/prismarine-viewer/pull/332

### CODE REVIEW

**Comment by extremeheat** [comment]
This is not how you do a release PR. You did not see the format for the previous ones. Also, I believe you are looking to bump minecraft-assets, not prismarine-viewer. Prismarine viewer doesn't support 1.19 at this time.

**Comment by KTibow** [comment]
ah okay
i did see the format for the previous ones, specifically https://github.com/PrismarineJS/prismarine-viewer/pull/242/files, was there something that i missed?

**Comment by extremeheat** [comment]
A release PR title must start with "Release" and contain a list of commits to the branch since the last release. Since there are no feature changes, the next release for prismarine-viewer should just be a patch bump.

**Comment by KTibow** [comment]
alright, i changed the pr to bump the patch number
it sounds like these other things have to happen before merging this pr
- merge https://github.com/PrismarineJS/prismarine-viewer/pull/332
- get someone with permissions to go to the workflows tab of https://github.com/PrismarineJS/node-minecraft-assets and create a version bump pr with the dispatch button
- possibly change some other stuff to make prismarine-viewer support 1.19.1?

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+### 1.23.1
+
+* Bump standard from 16.0.4 to 17.0.0
```
Comment (extremeheat) [inline HISTORY.md:3]: Don't need to include things that aren't relevant to the user, just changes that are relevant to API users

**Comment by extremeheat** [comment]
> get someone with permissions to go to the workflows tab of https://github.com/PrismarineJS/node-minecraft-assets and create a version bump pr with the dispatch button

You can make a release PR inside of minecraft-assets (see commit history for examples) and the workflow will handle the releases inside node-minecraft-assets. Then if #332 is passing it can be merged. 

That side, for this PR specifically to bump the current npm release to present master, it's currently failing on `canvas` module, this needs to be investigated.

File: HISTORY.md (hunk 1 above)
Comment (KTibow) [reply HISTORY.md:3]: so then remove everything except the 1.19 change, or is there something else relevant?

**Comment by KTibow** [comment]
The CI is failing because `prepare` is supposed to be ran before publish, but it also is running sometime before/in install. This could be fixed by changing it to prepublishOnly. Let me know if I should do that.
I also have to ask why package-lock.json is excluded from git. Its purpose is to help make CI work the same every time, it's not a local-only thing.

**Comment by KTibow** [comment]
okay, this pr should be ready to merge once you
- merge https://github.com/PrismarineJS/prismarine-viewer/pull/332 (and optionally add 1.19 to ci:
```
         - javaVersion: 17
           mcVersionIndex: 12
[...]
        CIRCLE_NODE_TOTAL: 13
```
- merge https://github.com/PrismarineJS/node-minecraft-assets/pull/24

**Comment by rom1504** [comment]
please update

**Comment by KTibow** [comment]
what here needs to be updated?

**Comment by rom1504** [comment]
there have been new changes https://github.com/PrismarineJS/prismarine-viewer/commits/master

**Comment by KTibow** [comment]
as in you need me to merge the latest changes in? that shouldn't be needed

**Comment by rom1504** [comment]
no, please update the history.md and the version number, there are new features so it should be 1.24.0

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@ jobs:
     - name: Install xi
       run: sudo apt-get install -y libxi-dev libglu1-mesa-dev
     - run: npm install
+    - run: npm run build
```
Comment (rom1504) [inline .github/workflows/ci.yml:38]: this seems wrong

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ jobs:
         with:
           node-version: 14
       - run: npm install
+      - run: npm run build
```
Comment (rom1504) [inline .github/workflows/publish.yml:16]: this seems wrong

File: HISTORY.md (hunk 1 above)
Comment (rom1504) [reply HISTORY.md]: fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (KTibow) [reply .github/workflows/ci.yml]: @rom1504 the `package` script breaks ci, [see earlier](https://github.com/PrismarineJS/prismarine-viewer/pull/358#issuecomment-1438897862)

**Pull request merged**